web-common/src/components/Portal.svelte

```diff
@@ -10,22 +9,21 @@
   let targetEntity;
 
   onMount(() => {
-    if (browser) {
-      portal = document.createElement("div");
-      portal.className = "portal";
-      if (!target) {
-        targetEntity = document.body;
-      } else {
-        targetEntity = document.querySelector(target);
-      }
-      targetEntity.appendChild(portal);
-      portal.appendChild(ref);
+    portal = document.createElement("div");
```

Comment:
we must remove the use of `$app/environment` since it is unnecessary (we have an SPA) and it's breaking Storybook.